internal/pkg/daemon/daemon_test.go

```diff
@@ -75,7 +75,7 @@ func TestIntegrationDaemon(t *testing.T) {
 	}()
 
 	// we can't register 2 metrics, but this sharing this should probably cause no problems
-	metrics := metrics.NewMetrics("/path/to/docker-compose.yml", "dummy", "test")
+	metrics := metrics.NewMetrics("/path/to/docker-compose.yml", "dummy", "test", "")
```

Comment:
i believe we don't check the `metrics` anywhere down the way (i mean in the tests), and providing an empty string to `NewMetrics()` creates a state that should not be possible as per `Config.ValidateAll()`

1/5 i would make it a non-empty string here, just to reflect the more likely state

i would also prefer if we could test the metrics in some way (and make sure it crashes if the `chain_id` is empty),  but i'm not sure if it' possible with the golang prometheus client and `promauto`